cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -49,25 +49,27 @@ case class CommandLineTool private(
   private [cwl] implicit val explicitWorkflowName = ParentName(id)
   private val inputNames = this.inputs.map(i => FullyQualifiedName(i.id).id).toSet
 
+  private[cwl] var parentWorkflowStep: Option[WorkflowStep] = None
```

Comment:
_Shame!_ 🔔 🔔 

No really-- can you leave a comment beside this var with the briefest of descriptions for future developers as to why the `var` is required?

Reply:
Yeah not my proudest moment but I did what I had to. 🙂 I'll add some docs.